Form/EventListener/ResizePolyFormListener.php

```diff
@@ -84,7 +85,12 @@ public function __construct(array $prototypes, array $options = array(), $allowA
                 $defaultType = $type;
             }
 
-            $typeKey = $type instanceof FormTypeInterface ? $type->getName() : $type;
+            $typeKey = $type;
+
+            if ($type instanceof FormTypeInterface) {
```

Comment:
I cant see the deprecation being triggered in the FormBuilder for this case nor an entry in 2.8's upgrading file regarding this - I assume the deprecation is handled deeper in the Form component?